### ISSUE TITLE
There are some security warnings in package.json, please fix

### ISSUE THREAD

**Issue by Jozdortraz** [draupnirstudio/Lacerta#14]
@miniwangdali

**Comment by miniwangdali** [comment]
```
  High            Arbitrary File Overwrite                                      
                                                                                
  Package         fstream                                                       
                                                                                
  Dependency of   @angular-devkit/build-angular [dev]                           
                                                                                
  Path            @angular-devkit/build-angular > node-sass > node-gyp >        
                  fstream                                                       
                                                                                
  More info       https://nodesecurity.io/advisories/886                        
                                                                                


                                                                                
  High            Arbitrary File Overwrite                                      
                                                                                
  Package         fstream                                                       
                                                                                
  Dependency of   @angular-devkit/build-angular [dev]                           
                                                                                
  Path            @angular-devkit/build-angular > node-sass > node-gyp > tar >  
                  fstream                                                       
                                                                                
  More info       https://nodesecurity.io/advisories/886                        
                                                                                


# Run  npm update tar --depth 4  to resolve 1 vulnerability
                                                                                
  High            Arbitrary File Overwrite                                      
                                                                                
  Package         tar                                                           
                                                                                
  Dependency of   @angular-devkit/build-angular [dev]                           
                                                                                
  Path            @angular-devkit/build-angular > node-sass > node-gyp > tar    
                                                                                
  More info       https://nodesecurity.io/advisories/803                        
                                                                                


                                                                                
                                 Manual Review                                  
             Some vulnerabilities require your attention to resolve             
                                                                                
          Visit https://go.npm.me/audit-guide for additional guidance           
                                                                                
                                                                                
  Moderate        Regular Expression Denial of Service                          
                                                                                
  Package         underscore.string                                             
                                                                                
  Patched in      >=3.3.5                                                       
                                                                                
  Dependency of   remarkable                                                    
                                                                                
  Path            remarkable > argparse > underscore.string                     
                                                                                
  More info       https://nodesecurity.io/advisories/745  
```
All above 4 vulnerabilities are not our direct dependencies. None of these package has an update to fix vulnerability issue yet. Update packages manually may break the dependencies. How would we solve it?

**Comment by Jozdortraz** [comment]
So just leave it and angular team may fix it later.
